src/index.ts

```diff
@@ -35,20 +54,42 @@ function test(description: Test['description'], testFunction: Test['testFunction
     tests.push(test)
 }
 
+function setTestSuiteOptions(newOptions: Partial<TestSuiteOptions>): void {
+    if (testsRunning) {
+        throw new Error(`Cannot set test suite options once tests are already running`)
+    }
+
+    if (testSuiteOptionsSet) {
+        throw new Error(`Can only set test suite options once`)
+    }
+
+    testSuiteOptionsSet = true
+
+    testSuiteOptions = {
+        ...testSuiteOptions,
+        ...newOptions,
+    }
+}
+
 // Run tests after synchronous test definitions happen. This means that tests MUST be defined synchronously.
 setImmediate(async () => {
     const startTime = Date.now()
     debugLog('running tests')
 
     testsRunning = true
-    await runTests()
+    await runTests(testSuiteOptions)
 
     debugLog(`(${Date.now() - startTime} ms) done running tests`)
 
     process.exitCode = errorCount > 0 ? 1 : 0
+
+    debugLog(`Sending kill signal SIGINT to self (process ID ${process.pid}). There are ${process.listenerCount(`SIGINT`)} current listeners for this signal.`)
+    process.kill(process.pid, `SIGINT`)  // exit the process, but allow handlers to catch the `SIGINT` and clean up as necessary. This is a gentler alternative to `process.exit()`.
```

Comment:
Maybe this should only happen if tests were cut off by the timeout. It could cause unexpected early exits, otherwise.

Reply:
Fixed in 05fbf11b134b58e8e2fc3909a1c0d14cd64bd1d1.